sideboard/lib/_profiler.py

```diff
@@ -0,0 +1,271 @@
+"""
+Adds profiling tools and a web interface for viewing profiling results.
+
+The Sideboard profiler borrows heavily from the `CherryPy profiler
+<https://github.com/cherrypy/cherrypy/blob/master/cherrypy/lib/profiler.py>`_,
+but with a few added features and nicer formatting.
+
+ * Adds the ability to sort results by different columns.
+ * Adds the ability to cleanup profile data files.
+ * Uses a better naming scheme for profile data files.
+ * Uses `cProfile` instead of `profile` for better performance.
+
+Profiling data can be collected using the @profile decorator on functions and
+methods. The profiling results can be viewed at http://servername/profile/.
+
+Good candidates for profiling are the outermost functions that generate your
+web pages, usually exposed as cherrypy endpoints via @cherrypy.expose::
+
+    import cherrypy
+    from sideboard.lib import profile
+
+    class Root(object):
+        @cherrypy.expose
+        @profile
+        def index(self):
+            # Create and return the index page
+            return '<html/>'
+
+
+But any regular function can be profiled using the @profile decorator::
+
+    from sideboard.lib import profile
+
+    @profile
+    def some_interesting_function():
+       # Do some stuff
+
+
+The following config options control how the profiler operates, see
+configspec.ini for more details::
+
+    [cherrypy]
+    profiling.on = boolean(default=True)
+    profiling.path = string(default="%(root)s/data/profiler")
+    profiling.aggregate = boolean(default=False)
+    profiling.strip_dirs = boolean(default=False)
```

Comment:
I think it might be clearer if we put example values rather than transcribing the actual definition of the config options, e.g.

```
[cherrypy]
profiling.on = True
profiling.strip_dirs = False
profiling.aggregate = False
profiling.path = "%(root)s/data/profiler"
```

Having talked to a lot of developers who aren't familiar with configobj, I think putting example values like that will be less confusing for newcomers than putting the ConfigObj optionb definition syntax here.